source/agora/consensus/protocol/Nominator.d

```diff
@@ -786,10 +838,21 @@ extern(D):
         if (data.tx_set.length == 0)
             assert(0, format!"Transaction set empty for slot %u"(slot_idx));
 
-        log.info("Externalized consensus data set at {}: {}", slot_idx, data);
+        log.info("Externalized consensus data set at {}: {}", slot_idx, prettify(data));
         try
         {
-            if (!this.ledger.onExternalized(data))
```

Comment:
So is `onExternalized` no longer used? Seems like it can be removed then?

Reply:
Removed from Ledger the unused onExternalized